class.photon.php

```diff
@@ -388,6 +389,24 @@ public static function filter_the_galleries( $galleries ) {
 		return $galleries;
 	}
 
+
+	/**
+	 * Runs the image widget through photon.
+	 *
+	 * @param array $instance Image widget instance data.
+	 * @return array
+	 */
+	function filter_the_image_widget( $instance ) {
+		if ( Jetpack::is_module_active( 'photon' ) && ! $instance['attachment_id'] && $instance['url'] ) {
+			jetpack_photon_url( $instance['url'], array(
```

Comment:
In the end of the day, `Jetpack_Photon::filter_the_content()` also uses `jetpack_photon_url()`. `$instance` provides the attachment id, you could just use that to pass width/height information to `jetpack_photon_url()`, no?